changes/745-djpetti.rst

```diff
@@ -0,0 +1,2 @@
+- Added support for `FrozenSet` members in dataclasses.
```

Comment:
```suggestion
Added support for `FrozenSet` members in dataclasses.
```
Should be just one line, I haven't build the processor for these changes yet, but it'll expect each change to become one bullet point.

Reply:
Done.